### ISSUE TITLE
How to build project?

### ISSUE THREAD

**Issue by tarlepp** [DataTorrent/malhar-angular-dashboard#138]
I have made fork of this repo and now I'm trying to build it, but I'm stuck... Basically I tried following as the docs says:

```
$ npm install -g grunt-cli
$ grunt
``` 

This gives me following error:

```
grunt-cli: The grunt command line interface. (v0.1.13)

Fatal error: Unable to find local grunt.

If you're seeing this message, either a Gruntfile wasn't found or grunt
hasn't been installed locally to your project. For more information about
installing and configuring grunt, please see the Getting Started guide:

http://gruntjs.com/getting-started
```

Also if I try ```npm test``` command all breaks because lo-dash is not found, below is the error log about this.

```
PhantomJS 1.9.8 (Linux) Factory: LayoutStorage the removeItem should call save FAILED
        ReferenceError: Can't find variable: _
            at iterator (/home/wunder/projects/malhar-angular-dashboard/src/components/models/LayoutStorage.js:168)
            at forEach (/home/wunder/projects/malhar-angular-dashboard/bower_components/angular/angular.js:326)
            at _addDefaultLayouts (/home/wunder/projects/malhar-angular-dashboard/src/components/models/LayoutStorage.js:169)
            at load (/home/wunder/projects/malhar-angular-dashboard/src/components/models/LayoutStorage.js:126)
            at LayoutStorage (/home/wunder/projects/malhar-angular-dashboard/src/components/models/LayoutStorage.js:54)
            at /home/wunder/projects/malhar-angular-dashboard/src/components/models/LayoutStorage.spec.js:672
        Error: spyOn could not find an object to spy upon for save()
            at /home/wunder/projects/malhar-angular-dashboard/src/components/models/LayoutStorage.spec.js:683
```

So how I can get things working?

**Comment by dlparker** [comment]
I have same problem. I have grunt-cli installed, so I tried to fix this by installing grunt locally with npm install grunt. Then I ran grunt,  but it was unable to load tasks:

dparker@camera1 ~/dev/malhar-angular-dashboard $ grunt
Loading "Gruntfile.js" tasks...ERROR
>> Error: Cannot find module 'load-grunt-tasks'
Warning: Task "default" not found. Use --force to continue.


Seems like a non-starter.
